database/user.go

```diff
@@ -631,3 +685,13 @@ func monthStart(subscribedUntil time.Time) time.Time {
 	d := now.AddDate(0, monthsDelta, daysDelta)
 	return time.Date(d.Year(), d.Month(), d.Day(), 0, 0, 0, 0, time.UTC)
 }
+
+// generateSub is a helper method that generates a UUID and encodes it in hex.
+func generateSub() (string, error) {
+	uid, err := uuid.New()
+	if err != nil {
+		build.Critical("Error during UUID creation:", err)
+		return "", err
+	}
+	return hex.EncodeToString(uid[:]), nil
```

Comment:
To avoid the error handling here you could do

Do you need a valid UUID here? Otherwise the following would work and not return error.
```go
var sub [16]byte
fastrand.Read(sub[:])
return hex.EncodeToString(sub[:])
```

Reply:
I prefer having an actual UUID.